### PR TITLE
feat: add initial equalized deploy command

### PR DESCRIPTION
**Pull request by PatrickMenoti**

### CODE REVIEW

**Review by maxwelbm — DISMISSED** [review]

File: pkg/cmd/deploy/scriptrunner.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+// Response represents the structure of the response from the API.
+type Response struct {
+	UUID  string    `json:"uuid"`
+	Image string    `json:"image"`
+	Start time.Time `json:"start"`
+}
+
+func callScript(token, id, secret, prefix, name string) (string, error) {
+	logger.Debug("Calling script runner api")
+	url := "https://stage-console.azion.com/api/template-engine/templates/92480a31-b88b-495b-8615-3ed5eff6314e/instantiate"
```
Comment (rodrigomaria) [inline pkg/cmd/deploy/scriptrunner.go:23]: Review this please.

File: pkg/api/storage/credentials.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+)
+
+type RequestCredentials struct {
+	sdk.S3CredentialCreate
+}
+
+func (c *Client) CreateCredentials(ctx context.Context, request RequestCredentials) (*sdk.ResponseS3Credential, error) {
+	logger.Debug("Creating s3 credentials ", zap.Any("name", request.Name))
+	req := c.apiClient.StorageAPI.StorageApiS3CredentialsCreate(ctx).S3CredentialCreate(request.S3CredentialCreate)
+	// req := c.apiClient.StorageAPI.StorageApiBucketsCreate(ctx).BucketCreate(request.BucketCreate)
```
Comment (rodrigomaria) [inline pkg/api/storage/credentials.go:19]: Comment //

**Review by PabloDiehl — APPROVED** [review]

File: env/prod [hunk 3]
```diff
@@ -1,5 +1,7 @@
 API_URL=https://api.azionapi.net
 AUTH_URL=https://sso.azion.com/api
 STORAGE_URL=https://api.azion.com
+CONSOLE=https://console.azion.com
```
Comment (rodrigomaria) [inline env/prod:4]: Can we use it as a secret too?

**Review by rodrigomaria — APPROVED** [review]

**Review by rodrigomaria — APPROVED** [review]

**Review by PabloDiehl — APPROVED** [review]

**Review by maxwelbm — DISMISSED** [review]

**Comment by maxwelbm** [comment]
Lgtm 🥷

**Pull request closed without merging**